x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -1467,7 +1467,13 @@ public enum Cap {
         /**
          * Support for the Present function
          */
-        FN_PRESENT;
+        FN_PRESENT,
+
+        /**
+         * Multivalued query parameters
+         */
+        QUERY_PARAMS_MULTI_VALUES(Build.current().isSnapshot());
```

Comment:
I'm thinking on not making this available just for snapshots, but would like to hear concerns about it

Reply:
I think the best practice is to remove snapshot in a followup PR right?